explorer/nodewatch/puller/client/NemClient.py

```diff
@@ -0,0 +1,41 @@
+from symbolchain.CryptoTypes import PublicKey
+
+from ..model.NodeInfo import NodeInfo
+from .BasicClient import BasicClient
+
+
+class NemClient(BasicClient):
+	"""Async client for connecting to a NEM node."""
+
+	async def chain_height(self):
+		"""Gets chain height."""
+
+		chain_height = await self.get('chain/height', 'height')
+		return int(chain_height)
```

Comment:
I'm guessing this (and in symbolclient below) will be Height?

Reply:
we could wrap it in pod from models